src/utils/__tests__/coordinates.test.ts

```diff
@@ -0,0 +1,119 @@
+import { describe, expect, it } from '@jest/globals'
+
+import { OPENLAYERS_PROJECTION, WSG84_PROJECTION } from '../../constants'
+import { CoordinatesFormat } from '../../fields/CoordinatesInput/constants'
+import { getCoordinates } from '../coordinates'
+
+describe('coordinates', () => {
+  it('getCoordinates Should get DMS coordinates for a dummy lon/lat', async () => {
```

Comment:
The test style respect the conventional unit testing wording that [Jest also use](https://jestjs.io/docs/tutorial-async): `it('should do something', () =>)`. The subject being the `describe('subject')` wrapper (you can have another nested describe for complex tests).